app/src/helpers/helpers.ts

```diff
@@ -185,3 +185,8 @@ export function removeUserAgentSpecifics(
     .replace(`Electron/${process.versions.electron} `, '')
     .replace(`${appName}/${appVersion} `, ' ');
 }
+
+// Removes formatting from a text
+export function getPlainTextWithoutFormatting(text: string): string {
```

Comment:
Unnecessary comment, and misleading function name. What about

```suggestion
export function cleanupPlainText(text: string): string {
```

?